content/artifacts/artifacts-helper/plugin.js

```diff
@@ -0,0 +1,241 @@
+import {
+    html,
+    render,
+    useState,
+    useLayoutEffect
+} from "https://unpkg.com/htm/preact/standalone.module.js";
+
+// Styles
+let Spacing = {
+    marginLeft: "6px",
+    marginRight: "6px",
+};
+let VerticalSpacing = {
+    marginBottom: "12px",
+};
+let HalfVerticalSpacing = {
+    marginBottom: "6px",
+};
+let Clickable = {
+    cursor: "pointer",
+    textDecoration: "underline",
+};
+let ActionEntry = {
+    marginBottom: "5px",
+    display: "flex",
+    justifyContent: "space-between",
+    color: "",
+};
+// Types
+/*
+const enum PlanetType {
+    PLANET = 0,
+    SILVER_MINE = 1,
+    RUINS = 2,
+    TRADING_POST = 3,
+    SILVER_BANK = 4
+}
+const enum PlanetType2 {
+    NOT_RUINS = 0,
+    NOT_PROSPECTED = 1,
+    PROSPECTING = 2,
+    READY_TO_FIND = 3,
+    FINDING = 4,
+    PROSPECTED_EXPIRED = 5,
+    NOT_READY_TO_PROSPECTED = 6,
+    FOUND = 7
+}
+*/
+var PlanetType;
+(function (PlanetType) {
+    PlanetType[PlanetType["PLANET"] = 0] = "PLANET";
+    PlanetType[PlanetType["SILVER_MINE"] = 1] = "SILVER_MINE";
+    PlanetType[PlanetType["RUINS"] = 2] = "RUINS";
+    PlanetType[PlanetType["TRADING_POST"] = 3] = "TRADING_POST";
+    PlanetType[PlanetType["SILVER_BANK"] = 4] = "SILVER_BANK";
+})(PlanetType || (PlanetType = {}));
+var PlanetType2;
+(function (PlanetType2) {
+    PlanetType2[PlanetType2["NOT_RUINS"] = 0] = "NOT_RUINS";
+    PlanetType2[PlanetType2["NOT_PROSPECTED"] = 1] = "NOT_PROSPECTED";
+    PlanetType2[PlanetType2["PROSPECTING"] = 2] = "PROSPECTING";
+    PlanetType2[PlanetType2["READY_TO_FIND"] = 3] = "READY_TO_FIND";
+    PlanetType2[PlanetType2["FINDING"] = 4] = "FINDING";
+    PlanetType2[PlanetType2["PROSPECTED_EXPIRED"] = 5] = "PROSPECTED_EXPIRED";
+    PlanetType2[PlanetType2["NOT_READY_TO_PROSPECTED"] = 6] = "NOT_READY_TO_PROSPECTED";
+    PlanetType2[PlanetType2["FOUND"] = 7] = "FOUND";
+})(PlanetType2 || (PlanetType2 = {}));
```

Comment:
Why not import the type from the `@darkforest_eth/types` package? That is why we published them

Reply:
I have no idea on how to import that, and even can't find the repository that generates the package on npmjs. Would you make some tips?

Reply:
Check out https://www.npmjs.com/package/@darkforest_eth/types Or on unpkg https://unpkg.com/@darkforest_eth/types@6.1.27/dist/index.js